news/_posts/2020-02-11-phpunit-security-issue-post-analysis.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "Security issue with PHPUnit, post-incident analysis"
+subtitle: "The mysterious development dependencies!"
+date:   2020-02-11 08:00:00
```

Comment:
Might be published tomorrow if review is late

Reply:
Hope it will be published today :D